connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -263,36 +266,37 @@ public void startConnect() {
      *
      * @param connName   the name of the connector
      * @param connConfig the intended configuration
-     * @throws IOException          if call to the REST api fails.
+     * @throws ConnectException if the configuration fails to be serialized
      * @throws ConnectRestException if REST api returns error status
      */
-    public void configureConnector(String connName, Map<String, String> connConfig) throws IOException {
+    public String configureConnector(String connName, Map<String, String> connConfig) {
```

Comment:
seems the return value is never used?

Reply:
Isn't a string returned on line 283?

Reply:
BTW, I think changing the return type from `void` to something else would be considered backward compatible, so this type of change is acceptable.